scripts/benchmark_compile_primitives.sh

```diff
@@ -3,9 +3,6 @@
 # Specify the base commands
 base_command="nargo compile --package "
 
-# Generate a sequence of arguments for all primitives
-arguments=("keccak256" "keccak256_100_times" "ecdsa_secp256k1" "compute_merkle_root_depth_4" "compute_merkle_root_depth_32" "verify_proof" "storage_proof_depth_8" "rsa")
-
```

Comment:
Does the script work with the arguments / packages removed?

Running it as is in the PR doesn't gather any benchmarking results on my end.

Reply:
I've treated those scripts not as self-sufficient entry points, but as helper scripts that are called from the main script and the main script sets the env variable. This allows to maintain a list in only one place instead of having it copied everywhere. If we want to maintain an ability to call them separately - we can store the list in a config file

Reply:
Gotcha, makes sense!